lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -8,6 +8,13 @@ class HostManager < TeracyDev::Config::Configurator
 
       def configure_common(settings, config)
         @plugins = settings['vagrant']['plugins'] ||= []
+
+        # get all eth networks
+        # then get the latest ip
+        @host_ip_command = "ip addr | grep eth | grep inet | cut -d/ -f1 | tail -1 | sed -e 's/^[ \t]*//' | cut -d' ' -f2"
```

Comment:
actually on some ubuntu version ethernet will use something like this `enp1s0`

Reply:
we can filter out the virtual interface on linux by `ls -la /sys/class/net/ | grep -v "virtual"` then handpick the physic interface and extract IP from it, then the next problem is determining which IP is public/private/default

Reply:
Because you `grep eth` the network interface name is not always eth0 eth1, depend on kernel version it can jump to enp1s0, enp1s2.
@hoatle can you confirm about that on our VM, I believed that you saw enp1s0 before.
example.
https://www.itzgeek.com/how-tos/mini-howtos/change-default-network-name-ens33-to-old-eth0-on-ubuntu-16-04.html
https://askubuntu.com/questions/767786/changing-network-interfaces-name-ubuntu-16-04

Reply:
related issues:
- https://github.com/teracyhq/dev/issues/434#issuecomment-393748057
- https://github.com/teracyhq/dev/issues/433

Reply:
I changed the box version to `Ubuntu 18.04.1 LTS`, still working normally, please provide a version that produce that error

Reply:
As you wish.
```
  vm:
    box: bento/ubuntu-16.04
    box_version: 2.3.8
```

Reply:
thank you very much for the specific version, it produced now, fixing it